internal/service/databaseuser/resource_database_user_migration_test.go

```diff
@@ -286,21 +286,22 @@ func TestAccMigrationConfigRSDatabaseUser_WithRoles(t *testing.T) {
 func TestAccMigrationConfigRSDatabaseUser_WithScopes(t *testing.T) {
 	var (
 		resourceName = "mongodbatlas_database_user.test"
-		username     = acctest.RandomWithPrefix("test-acc-user-")
-		password     = acctest.RandomWithPrefix("test-acc-pass-")
+		username     = acctest.RandomWithPrefix("test-acc-user")
```

Comment:
curious why you removed `-`?

Reply:
because it's added already by RandomWithPrefix so here we have --

Reply:
oh ok, thanks for catching that.